library/src/main/res/values/dimens.xml

```diff
@@ -1,8 +1,8 @@
 <!-- Copyright 2012 Square, Inc. -->
 <resources>
   <dimen name="calendar_day_headers_paddingbottom">4dp</dimen>
-  <dimen name="calendar_month_topmargin">16dp</dimen>
-  <dimen name="calendar_month_title_bottommargin">4dp</dimen>
+  <dimen name="calendar_month_title_paddingtop">16dp</dimen>
```

Comment:
I don't think we should rename these.  It's pretty likely that people are overriding these values just by defining their dimens with the same name, and if we change the name, their customizations will silently fail.

Reply:
I didn't think about it. Nice catch! I am changing now.